modules/base/tool.go

```diff
@@ -465,41 +465,41 @@ func Subtract(left interface{}, right interface{}) interface{} {
 	var rleft, rright int64
 	var fleft, fright float64
 	var isInt = true
-	switch left := left.(type) {
+	switch v := left.(type) {
 	case int:
-		rleft = int64(left)
+		rleft = int64(v)
 	case int8:
-		rleft = int64(left)
+		rleft = int64(v)
 	case int16:
-		rleft = int64(left)
+		rleft = int64(v)
 	case int32:
-		rleft = int64(left)
+		rleft = int64(v)
 	case int64:
-		rleft = left
+		rleft = v
 	case float32:
-		fleft = float64(left)
+		fleft = float64(v)
 		isInt = false
 	case float64:
-		fleft = left
+		fleft = v
 		isInt = false
 	}
 
-	switch right := right.(type) {
+	switch v := right.(type) {
```

Comment:
Could we not refactor this further to get rid of the code duplication completely?